### DISCUSSION TITLE
Chinese input display problem

### DISCUSSION THREAD

**Discussion by aiden-xiang** [electerm/electerm#3734] [General]
Hi,

I am using electerm 1.51.3 version. When I use the Chinese input method to input a Chinese character in a Chinese string, the last Chinese character of the Chinese sentence is displayed，the Chinese character I want to enter should be displayed. However，other terminal tools didn't find this issue.
Can this problem would be solved?

Electerm Version: 1.51.3
Laptop: win 7

Regards.

**Comment by zxdong262** [comment]
Might be a win7 issue, for it to work in win7, have to build with old release of electron, in win11 it works fine
